packages/builder/src/components/deploy/RevertModalVersionSelect.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { API } from "api"
+  import clientVersions from "./clientVersions.json"
+  import { appStore } from "stores/builder"
+  import { Select } from "@budibase/bbui"
+
+  export let revertableVersion
+  $: appId = $appStore.appId
+
+  const handleChange = e => {
+    const value = e.value
```

Comment:
This should be `e.detail`. Reverting without this throws an error.

Reply:
My bad, I had changed this, no idea how it went back to this.